### PR TITLE
AC-657 added About section

### PR DESCRIPTION
**Pull request by MasterBlaster99**
### Description of what i have changed 
Added the About section page which redirects to the about page of OpenMRS website.
### Issue I worked on 
JIRA Issue : https://issues.openmrs.org/browse/AC-657
## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [ ] My pull request is **based on the latest changes** of the master branch.

Screenshot (521)

### CODE REVIEW

**Comment by MasterBlaster99** [comment]
Issue is assigned to me

File: openmrs-client/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -69,6 +69,11 @@
             android:label="Introduction"
             android:theme="@style/AppTheme">
         </activity>
+        <activity
+            android:name=".activities.AboutActivity"
+            android:label="about"
```
Comment (rishabh-997) [inline openmrs-client/src/main/AndroidManifest.xml:74]: Add this label in string resources and use it directly here.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java [hunk 2]
```diff
@@ -219,6 +219,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 Observable<List<LocationEntity>> observableList = new LocationDAO().getLocations();
                 observableList.subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread()).subscribe(getLocationList());
                 return true;
+            case R.id.actionAbout:
+                Intent intent =new Intent(getApplicationContext(),AboutActivity.class);
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java:223]: Indent in properly.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package org.openmrs.mobile.activities;
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java:1]: Add license code here

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import org.openmrs.mobile.R;
+
+public class AboutActivity extends AppCompatActivity {
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java:13]: Extend it to ACBaseActivity instead.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_about);
+        TextView tv = findViewById(R.id.moreAbout);
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java:19]: use proper names for variables related to what they do

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+     tv.setOnClickListener(new View.OnClickListener() {
+         @Override
+         public void onClick(View v) {
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java:23]: Use lambda function here

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+             Intent intent=new Intent(Intent.ACTION_VIEW);
+             intent.setData(Uri.parse("https://openmrs.org/about/"));
+             startActivity(intent);
+         }
+     });
+
+
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java:30]: Why this space?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_about);
+        TextView tv = findViewById(R.id.moreAbout);
+
+     tv.setOnClickListener(new View.OnClickListener() {
+         @Override
+         public void onClick(View v) {
+             Intent intent=new Intent(Intent.ACTION_VIEW);
+             intent.setData(Uri.parse("https://openmrs.org/about/"));
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.java:25]: Store this in ApplicationConstants

File: openmrs-client/src/main/res/menu/basic_menu.xml [hunk 9]
```diff
@@ -43,4 +43,9 @@
         android:orderInCategory="100"
         android:title="@string/action_logout"
         app:showAsAction="never" />
+    <item
+        android:id="@+id/actionAbout"
+        android:orderInCategory="100"
+        android:title="About"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/menu/basic_menu.xml:49]: Add this in string resource file

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 10]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:1]: add license

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+    android:layout_height="match_parent"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".activities.AboutActivity">
+
+    <ImageView
+        android:id="@+id/openmrsLogoAbout"
+        android:layout_width="110dp"
+        android:layout_height="100dp"
+
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:13]: why this space

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_centerHorizontal="true"
+        android:layout_marginLeft="70dp"
+        android:layout_marginTop="60dp"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:16]: Align it properly

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+    <TextView
+        android:id="@+id/paragraph1"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/openmrsLogoAbout"
+        android:layout_marginTop="40dp"
+        android:layout_marginLeft="30dp"
+        android:layout_marginRight="30dp"
+        android:textSize="22sp"
+        android:text="OpenMRS is a global community of truly dedicated, talented, and generous contributors who build and maintain the OpenMRS platform and other, foundational OpenMRS technical products. "
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:44]: store it in string resouce

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/textView3"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="30dp"
+        android:layout_marginTop="15dp"
+        android:layout_marginRight="30dp"
+        android:text="OpenMRS is a platform that countries and implementers use to create a customized EMR system in response to actual needs on the ground. "
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:55]: again

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_marginTop="40dp"
+        android:layout_marginLeft="30dp"
+        android:layout_marginRight="30dp"
+        android:textSize="22sp"
+        android:text="OpenMRS is a global community of truly dedicated, talented, and generous contributors who build and maintain the OpenMRS platform and other, foundational OpenMRS technical products. "
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/textView3"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:49]: give it proper name

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+        android:fontFamily="@font/montserrat"
+        android:text="@string/organization_name"
+        android:textColor="@color/black"
+        android:textSize="30sp"
+        app:layout_constraintBottom_toBottomOf="@+id/openmrsLogoAbout"
+        app:layout_constraintStart_toEndOf="@+id/openmrsLogoAbout"
+        app:layout_constraintTop_toTopOf="@+id/openmrsLogoAbout" />
+
+    <TextView
+        android:id="@+id/paragraph1"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:36]: give proper name

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_marginTop="15dp"
+        android:layout_marginRight="30dp"
+        android:text="OpenMRS is a platform that countries and implementers use to create a customized EMR system in response to actual needs on the ground. "
+        android:textSize="22sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/paragraph1" />
+
+    <TextView
+        android:id="@+id/moreAbout"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:62]: give proper name

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="60dp"
+        android:layout_marginEnd="40dp"
+        android:layout_marginRight="40dp"
+        android:shadowColor="@color/neutral_green"
+        android:text="more >>"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:69]: again in string resouce please

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+        android:textColor="@color/neutral_green"
+        android:textSize="23sp"
+        android:textStyle="bold|italic"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:72]: dont do this, use standard fonts

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView3" />
+
+
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:76]: why this space

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Please go through codes of other activities to get the idea of conventions. Also, use kotlin instead of java. Do this, then I will give more reviews.

**Comment by rishabh-997** [comment]
And don't change the template of the description of this PR, check other PR's and follow the convention instead of erasing everything.

**Comment by MasterBlaster99** [comment]
> Please go through codes of other activities to get the idea of conventions. Also, use kotlin instead of java. Do this, then I will give more reviews.

Yes sir , I'll work on all of these

**Comment by MasterBlaster99** [comment]
> Please go through codes of other activities to get the idea of conventions. Also, use kotlin instead of java. Do this, then I will give more reviews.
I have made the changes 
should i commit

**Comment by MasterBlaster99** [comment]
> Please go through codes of other activities to get the idea of conventions. Also, use kotlin instead of java. Do this, then I will give more reviews.

I have made the changes , also used kotlin 
should i make 1 more commit ?

**Comment by MasterBlaster99** [comment]
Sir , I have made the changes 
should i commit the changes

**Comment by rishabh-997** [comment]
Yes, I dont think there is any other way to check the new changes :laughing: Also, please correct the description, it is still not matching the template

**Comment by MasterBlaster99** [comment]
> Yes, I dont think there is any other way to check the new changes 😆 Also, please correct the description, it is still not matching the template
Yes 😅, I have made the changes @rishabh-997 please review

**Comment by MasterBlaster99** [comment]
> Yes, I dont think there is any other way to check the new changes 😆 Also, please correct the description, it is still not matching the template

Yes 😅, I have made the changes @rishabh-997 please review

File: openmrs-client/src/main/AndroidManifest.xml [hunk 21]
```diff
@@ -1,18 +1,6 @@
...
-    ~ http://license.openmrs.org
-    ~
-    ~ Software distributed under the License is distributed on an "AS IS"
-    ~ basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
-    ~ License for the specific language governing rights and limitations
-    ~ under the License.
-    ~
-    ~ Copyright (C) OpenMRS, LLC.  All Rights Reserved.
-    -->
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (LuGO0) [inline openmrs-client/src/main/AndroidManifest.xml:1]: did you intentionally remove the copyright ?

File: openmrs-client/src/main/res/values/strings.xml [hunk 22]
```diff
@@ -509,4 +509,11 @@
...
     <string name="start">Start:</string>
     <string name="end">End:</string>
+
+    <!-- About Activity -->
+    <string name="action_about_activity">about</string>
+    <string name="action_about">About</string>
+    <string name="about_openmrs_1">OpenMRS is a global community of truly dedicated, talented, and generous contributors who build and maintain the OpenMRS platform and other, foundational OpenMRS technical products.</string>
+    <string name="about_openmrs_2">OpenMRS is a platform that countries and implementers use to create a customized EMR system in response to actual needs on the ground.</string>
+    <string name="more_about_openmrs"><![CDATA[more >>]]></string>
 </resources>
```
Comment (LuGO0) [inline openmrs-client/src/main/res/values/strings.xml:519]: did you add translations?

**Review by LuGO0 — CHANGES_REQUESTED** [review]

File: openmrs-client/src/main/AndroidManifest.xml (hunk 21 above)
Comment (MasterBlaster99) [reply openmrs-client/src/main/AndroidManifest.xml:1]: oh sorry ! i removed it by mistake, i'll fix it

File: openmrs-client/src/main/res/values/strings.xml (hunk 22 above)
Comment (MasterBlaster99) [reply openmrs-client/src/main/res/values/strings.xml:519]: No, I'll add

File: openmrs-client/src/main/AndroidManifest.xml (hunk 21 above)
Comment (MasterBlaster99) [reply openmrs-client/src/main/AndroidManifest.xml:1]: I added the copyright

File: openmrs-client/src/main/res/values/strings.xml (hunk 22 above)
Comment (MasterBlaster99) [reply openmrs-client/src/main/res/values/strings.xml:519]: @LuGO0 i added translations

File: openmrs-client/src/main/AndroidManifest.xml (hunk 21 above)
Comment (MasterBlaster99) [reply openmrs-client/src/main/AndroidManifest.xml:1]: I added the copyright 
@LuGO0 please review

**Comment by MasterBlaster99** [comment]
I have made the changes 
@rishabh-997 , @LuGO0 please review

**Comment by LuGO0** [comment]
build is failing look up to it

**Comment by MasterBlaster99** [comment]
> build is failing look up to it

**Comment by MasterBlaster99** [comment]
> build is failing look up to it

Sir, But the gradle build is running successfully and aslo the app is working fine after changes

**Comment by MasterBlaster99** [comment]
> build is failing look up to it

Yes I am looking

**Comment by MasterBlaster99** [comment]
> build is failing look up to it

@LuGO0  now the build is successful

**Comment by LuGO0** [comment]
> > build is failing look up to it
> 
> now the build is successful
> @LuGO0 please reviw

@rishabh-997 I think we can should share the figma designs for other screens to @MasterBlaster99 so that the about section is also consistent with other screens.

**Comment by LuGO0** [comment]
https://www.figma.com/file/LzEJgldSbhdIuBiFQkePf0/OpenMRS-android-designs?node-id=0%3A1 here it is

**Comment by MasterBlaster99** [comment]
> https://www.figma.com/file/LzEJgldSbhdIuBiFQkePf0/OpenMRS-android-designs?node-id=0%3A1 here it is, try and make your design consistent with the figma link.

Yes , i am doing

**Comment by MasterBlaster99** [comment]
> https://www.figma.com/file/LzEJgldSbhdIuBiFQkePf0/OpenMRS-android-designs?node-id=0%3A1 here it is, try and make your design consistent with the figma link.

Yes , i am trying

**Comment by MasterBlaster99** [comment]
> https://www.figma.com/file/LzEJgldSbhdIuBiFQkePf0/OpenMRS-android-designs?node-id=0%3A1 here it is, try and make your design consistent with the figma link.

Yes , i am doing

**Comment by rishabh-997** [comment]
> Yes, I am doing

Post the design on Jira issues for review. Standard font and colors are available in the figma link

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt [hunk 23]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.mobile.activities
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt:15]: Move this activity to ContactUs Package. Try to maintain the architecture.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt [hunk 24]
```diff
@@ -0,0 +1,32 @@
...
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import org.openmrs.mobile.utilities.ApplicationConstants
+import kotlinx.android.synthetic.main.activity_about.*
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt:21]: Don't use star imports as this is not a good practice.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt [hunk 25]
```diff
@@ -0,0 +1,32 @@
...
+import org.openmrs.mobile.R
+
+class AboutActivity : ACBaseActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_about)
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt:27]: Use ActivityBinding instead as done in other activities.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt [hunk 26]
```diff
@@ -0,0 +1,32 @@
...
+        OpenMRSWebsiteTextView.setOnClickListener { intent = Intent(Intent.ACTION_VIEW)
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt:28]: Use toggle case to name variables. and send intent to next line.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt [hunk 27]
```diff
@@ -0,0 +1,32 @@
...
+            intent.setData(Uri.parse(ApplicationConstants.ABOUT_OPENMRS_URL))
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/AboutActivity.kt:29]: Combine these in a single line.
val intent = Intent(Intent.ACTION_VIEW, Uri.parse(getString(ApplicationConstants.ABOUT_OPENMRS_URL)))

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 28]
```diff
@@ -0,0 +1,87 @@
...
+        android:layout_marginTop="15dp"
+        android:layout_marginRight="30dp"
+        android:text="@string/about_openmrs_2"
+        android:textSize="22sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/aboutOpenMRSTextView1" />
+
+    <TextView
+        android:id="@+id/OpenMRSWebsiteTextView"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:76]: again, use toggle case starting with lowercase

File: openmrs-client/src/main/res/menu/basic_menu.xml [hunk 29]
```diff
@@ -43,4 +43,9 @@
         android:orderInCategory="100"
         android:title="@string/action_logout"
         app:showAsAction="never" />
+    <item
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/menu/basic_menu.xml:46]: LogOut option is always at the bottom

File: openmrs-client/src/main/res/values-hi/strings.xml [hunk 30]
```diff
@@ -470,4 +470,9 @@
     <string name="allergy_already_selected">पहले से ही चयनित एलर्जी!</string>
     <string name="start">शुरू:</string>
     <string name="end">समाप्त:</string>
+    <string name="action_about_activity">के बारे में</string>
+    <string name="action_about">के बारे में</string>
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/values-hi/strings.xml:474]: How is this a translation for "about"? Use somewhat more meaningful.

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Still a lot of changes required, do this meanwhile

**Comment by MasterBlaster99** [comment]
> > Yes, I am doing
> 
> Post the design on Jira issues for review. Standard font and colors are available in the figma link

ok

**Comment by MasterBlaster99** [comment]
> > Yes, I am doing
> 
> Post the design on Jira issues for review. Standard font and colors are available in the figma link

i have posted the design on jira issues [check it here](https://issues.openmrs.org/issues/?jql=project%20%3D%20AC%20AND%20resolution%20%3D%20Unresolved%20AND%20text%20~%20%22about%20section%22%20ORDER%20BY%20status%20DESC%2C%20cf%5B10020%5D%20ASC%2C%20priority%20DESC%2C%20updated%20DESC)

**Comment by MasterBlaster99** [comment]
I have made the requested changes 
please review the design so i can commit the changes

**Comment by MasterBlaster99** [comment]
I have made the changes @rishabh-997 , @LuGO0

**Comment by MasterBlaster99** [comment]
@rishabh-997 , @LuGO0  i have made the changes

**Comment by MasterBlaster99** [comment]
@rishabh-997 , @LuGO0  please review !!

**Comment by rishabh-997** [comment]
> @rishabh-997 , @LuGO0 please review !!

Can you add a combined screenshot in
1. Normal
2. Dark mode
3. Hindi mode

**Comment by MasterBlaster99** [comment]
> > @rishabh-997 , @LuGO0 please review !!
> 
> Can you add a combined screenshot in
> 
> 1. Normal
> 2. Dark mode
> 3. Hindi mode

### 1.Normal
WhatsApp Image 2021-01-31 at 1 33 30 PM (1)

### 2. Hindi mode 
WhatsApp Image 2021-01-31 at 1 33 30 PM

**Comment by MasterBlaster99** [comment]
Normal mode and Dark mode are exactly same, as in all other activities

**Comment by MasterBlaster99** [comment]
> > @rishabh-997 , @LuGO0 please review !!
> 
> Can you add a combined screenshot in
> 
> 1. Normal
> 2. Dark mode
> 3. Hindi mode


### 1. Normal 
WhatsApp Image 2021-01-31 at 1 33 30 PM (1)

### 2. Dark mode 
darkOpenMRS


### 3. Hindi mode 
WhatsApp Image 2021-01-31 at 1 33 30 PM

**Comment by MasterBlaster99** [comment]
@LuGO0 , @rishabh-997  please review !!

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/community/contact/AboutActivity.kt [hunk 31]
```diff
@@ -0,0 +1,62 @@
...
+        binding = ActivityAboutBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val actionBar = supportActionBar
+        if (actionBar != null) {
+            actionBar.elevation = 0f
+            actionBar.setDisplayHomeAsUpEnabled(true)
+        }
+
+        binding.mOREaBOUToPENMRSbUTTON.setOnClickListener {
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/community/contact/AboutActivity.kt:40]: Use standard naming conventions as in other activities. This looks too weird :laughing:

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 32]
```diff
@@ -0,0 +1,93 @@
...
+        android:fontFamily="@font/montserrat"
+        android:text="@string/organization_name"
+        android:textColor="?android:attr/textColorPrimary"
+        android:textSize="22sp"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/openmrsLogo" />
+
+    <TextView
+        android:id="@+id/aBOUToPENMRStEXTvIEW1"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:49]: use a proper naming convention.

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+        android:layout_marginLeft="30dp"
+        android:layout_marginRight="30dp"
+        android:textSize="22sp"
+        android:textColor="?android:attr/textColorPrimary"
+        android:text="@string/about_openmrs_1"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/aBOUToPENMRStEXTvIEW2"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:63]: use a proper naming convention.

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 34]
```diff
@@ -0,0 +1,93 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="30dp"
+        android:layout_marginTop="15dp"
+        android:layout_marginRight="30dp"
+        android:text="@string/about_openmrs_2"
+        android:textSize="22sp"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:70]: make it around 18-20sp

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 35]
```diff
@@ -0,0 +1,93 @@
...
+        android:textColor="?android:attr/textColorPrimary"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/aBOUToPENMRStEXTvIEW1" />
+
+    <TextView
+        android:id="@+id/mOREaBOUToPENMRSbUTTON"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:77]: use a proper naming convention.

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Also, take a look at the stack hierarchy. moving from settings->about->setting->about->so...on creates a piling stack.

**Comment by MasterBlaster99** [comment]
> Also, take a look at the stack hierarchy. moving from settings->about->setting->about->so...on creates a piling stack.

should i make only "settings" visible in the menu of about activity ?

**Comment by MasterBlaster99** [comment]
> Also, take a look at the stack hierarchy. moving from settings->about->setting->about->so...on creates a piling stack.

I did not understood what i exactly have to do here 🤔

**Comment by rishabh-997** [comment]
> I did not understand what I exactly have to do here

https://user-images.githubusercontent.com/45125121/106630090-8591d880-65a1-11eb-81bc-c67f46e07529.mp4

**Comment by MasterBlaster99** [comment]
> > I did not understand what I exactly have to do here
> 
>  WhatsApp.Video.2021-02-02.at.9.54.07.PM.mp4

Oh! Got it , thank you

**Comment by MasterBlaster99** [comment]
@rishabh-997 Done !

**Comment by MasterBlaster99** [comment]
@rishabh-997 , @LuGO0  please review !!

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 36]
```diff
@@ -0,0 +1,93 @@
...
+        android:textSize="20sp"
+        android:textColor="?android:attr/textColorPrimary"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/aboutOpenmrsTextView" />
+
+    <TextView
+        android:id="@+id/moreAboutOpenmrsButton"
+        android:layout_width="wrap_content"
+        android:layout_height="42dp"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:79]: use wrap content and also use gravity to make the text at the centre of the button.

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 37]
```diff
@@ -0,0 +1,93 @@
...
+
+    <TextView
+        android:id="@+id/aboutOpenmrsTextView2"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="35dp"
+        android:layout_marginTop="15dp"
+        android:layout_marginRight="35dp"
+        android:text="@string/about_openmrs_2"
+        android:textSize="20sp"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:70]: This too.

File: openmrs-client/src/main/res/layout/activity_about.xml [hunk 38]
```diff
@@ -0,0 +1,93 @@
...
+
+    <TextView
+        android:id="@+id/aboutOpenmrsTextView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/organization_name"
+        android:layout_marginTop="30dp"
+        android:layout_marginLeft="35dp"
+        android:layout_marginRight="35dp"
+        android:textSize="20sp"
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/layout/activity_about.xml:56]: This still looks too big for a descriptive text. make this 18sp

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
LGTM. just these 2 changes and looks fine.

**Comment by MasterBlaster99** [comment]
> LGTM. just these 2 changes and looks fine.

@rishabh-997 Made the Changes, Please Review !!

**Review by rishabh-997 — APPROVED** [review]
LGTM!!!

**Pull request merged**